app/data/dbConn.py

```diff
@@ -10,12 +13,12 @@ def connect():
         # SQLite objects created in a thread can only be used in that same thread.
         # tech_same_thread=False repaired it
         connection = sqlite3.connect(app_config.DATABASE_PATH, check_same_thread=False)
-        print(sqlite3.version)
 
+        db_logger.debug("Successfully connected to database")
         return connection
 
     except Error as e:
-        print(e)
+        db_logger.critical("Connection to database failed", exc_info=True)        
```

Comment:
if that happens you should close app probably
in future add some retry connection feature